spec/features/blackout_spec.rb

```diff
@@ -41,6 +41,34 @@
     end
   end
 
+  describe 'recurring blackout' do
```

Comment:
Can we move the two contexts above to their own `describe` block for `regular blackout`? Just to be consistent.